build.gradle

```diff
@@ -17,7 +17,7 @@ repositories {
 dependencies {
     testImplementation 'junit:junit:4.12'
 
-    implementation 'com.vonage:client:6.0.0'
+    implementation 'com.vonage:client:6.1.0'
```

Comment:
I'd change this to the 6.x range, to avoid having to constantly update this.

Reply:
fixed